scripts/draw_container_get_properties/draw_container_get_properties.gml

```diff
@@ -134,6 +134,11 @@ function draw_container_get_properties(data)
 		data.paddingBottom = data.padding;
 	}
 	
+	if (!variable_struct_exists(data, "decorators"))
+	{
+		data.decorators = [];
+	}
+	
```

Comment:
For simplicity, we should simply call this `emphasisColor`, and default to `c_black` (inverse of `textColor`)

Reply:
Done (with `c_orange` as the default)